package.json

```diff
@@ -41,28 +42,28 @@
   "homepage": "https://github.com/stashenergy/react-native-msal#readme",
   "dependencies": {
     "@azure/msal-browser": "^2.3.4",
-    "@expo/config-plugins": "~7.2.2"
+    "@expo/config-plugins": "~8.0.8"
```

Comment:
are you having the same issue where the build folder is not present? i have added the yarn build step to the prepare script as well, it might be an issue with npm using prepare and yarn using prepack

Reply:
I think that is the case yes. With NPM, I can't even install the newest commit though, so can't verify